docs/using-cinder-csi-plugin.md

```diff
@@ -21,6 +21,11 @@ Check [kubernetes CSI Docs](https://kubernetes-csi.github.io/docs/) for flag det
 
 ### Deploy
 
+
+ToDo  ---> https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-controller-manager-with-kubeadm.md#steps
```

Comment:
@johscheuer I suggest you to separate changes for controller manager and CSI, Please create a new PR for CSI changes. It is creating lots of confusion here and CSI CI failure.

Reply:
I separated the changes for CCM and CSI (CSI MR will follow soon).